### PR TITLE
Fix issue 258

### LINKED ISSUE #258: [#256] Create error flash message under Ui components

### PR DESCRIPTION
**Pull request by shin5446**
## Content
As the title says
## Related Issue
fixed #258 

## notes
Since the flash message components were spread around in various places, I set it up so that it would be possible to modify them by changing one part.

### CODE REVIEW

File: src/resources/js/Components/Pages/Main/Index.jsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ function Index({ auth, events, flash, tags }) {
 
     return (
         <div className="flex h-screen flex-col">
+            {/* TODO: Use GeneralLayout */}
```
Comment (shin5446) [inline src/resources/js/Components/Pages/Main/Index.jsx:43]: Let's fix this one at some point.

**Review by akaka0039 — COMMENTED** [review]
Great code!!!

File: src/resources/js/Components/Ui/FlashMessage.jsx
Comment (akaka0039) [inline src/resources/js/Components/Ui/FlashMessage.jsx:1]: I'm not sure, but can't see any backend color on flash message.
I found what it cause this cases might be [This](https://tailwindcss.com/docs/content-configuration#dynamic-class-names)
Could you follow the document and fix the code? Or another PR is also fine for me.

File: src/resources/js/Components/Pages/Main/Index.jsx (hunk 1 above)
Comment (akaka0039) [reply src/resources/js/Components/Pages/Main/Index.jsx:43]: I forget it haha
I've created [tickets](https://github.com/akaka0039/Global_Developer_site/issues/285)

File: src/resources/js/Components/Ui/FlashMessage.jsx
Comment (shin5446) [reply src/resources/js/Components/Ui/FlashMessage.jsx:1]: Humm, In my environment, it is working well...

**Review by akaka0039 — APPROVED** [review]

**Pull request merged**